eZ/Publish/API/Repository/Tests/RoleServiceTest.php

```diff
@@ -1708,7 +1745,7 @@ public function testGetRoleAssignments()
 
         /* END: Use Case */
 
-        $this->assertEquals(1, count($roleAssignments));
+        $this->assertEquals(2, count($roleAssignments));
```

Comment:
@pspanja Well, the test is technically correct, if incomplete :) Ok, I can test both objects.